examples/clustering/plot_dtw_boss.py

```diff
@@ -0,0 +1,104 @@
+"""
+=========================================
+Time Series Clustering with DTW and BOSS
+=========================================
+
+This example shows the differences between various metrics
+related to time series clustering. Besides the Euclidean distance,
+:func:`pyts.metrics.dtw` and :func:`pyts.metrics.boss` are considered to
+analyze the :func:`pyts.datasets.make_cylinder_bell_funnel` dataset.
+In contrast to reference [1], DTW-based clustering shows the best results
+here - even when repeated over multiple seeds. The reason for this could be
+differences in the experimental setup, e.g., the hyperparameters of the methods
+or the number of considered samples. Depending on the time series structure, a
+suitable metric should be chosen: While the Euclidean distance and DTW are
+sensitive to time-dependent events, clustering with BOSS does not consider the
+temporal component outside the sliding window.
+
+References
+----------
+
+[1] P. Schäfer, "The BOSS is concerned with time series classification
+    in the presence of noise". Data Mining and Knowledge Discovery,
+    29(6), 1505-1530 (2015).
```

Comment:
It does not display well. I think that it takes the `P.` as a symbol of enumeration (similar to an enumerate environment in LaTeX). I had this issue with `L. Breiman`, and replacing it with `Leo Breiman` fixed it. You may try using the full first name (Patrick) to fix it.

Reply:
Couldn't see exactly how the error shows up in my browser. I have simply included your suggestion. Please check again if your problem is solved this way.

Reply:
<img width="641" alt="Capture d’écran 2022-06-16 à 10 50 01" src="https://user-images.githubusercontent.com/17053934/174032112-601379cb-f41c-43d1-a4c1-1b03dd484838.png">

I see a line break, but I don't update my browser very often, so it may explain this. It's not really important anyway, thanks for trying out!